README.md

```diff
@@ -37,9 +37,13 @@ Inspired by [rustup](https://github.com/rust-lang-nursery/rustup.rs).
 
 ## Installation
 
-Just place the `ghcup` shell script into your PATH anywhere
+Just place the `ghcup` shell script into your `PATH` anywhere
 (preferably `~/.local/bin`).
 
+```
+curl https://raw.githubusercontent.com/hasufell/ghcup/master/ghcup > ~/.local/bin/ghcup && chmod 755 ~/.local/bin/ghcup
```

Comment:
I prefer to be a bit more verbose here, so people are forced to read the command ;)

E.g.:

```sh
mkdir -p ~/.local/bin/ghcup
curl https://raw.githubusercontent.com/hasufell/ghcup/master/ghcup > ~/.local/bin/ghcup
chmod +x ~/.local/bin/ghcup
```
```

Reply:
done